docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "2"
+services:
+  backup-service:
+    container_name: backup-service
+#    restart: always
+    build: .
+    ports:
+      - "8000:8000"
+    links:
+      - mongo
+  mongo:
+    container_name: mongo
+    image: mongo
```

Comment:
Please use at least a major version tag on the image, to avoid surprises down the line